Documentation/BuildInstructionsWindows.md

```diff
@@ -51,3 +51,8 @@ following command in an elevated PowerShell session: \
 You may have to reboot after enabling the WHPX feature.
 
 Afterwards you can start the VM with `Meta/serenity.sh run` as usual.
+
+#### Runtime issues
+
+1. When running `Meta/serenity.sh run` on Windows you're receiving error `qemu-system-i386.exe: -s: Failed to bind socket: Input/output error`. The `-s` does not work, and causes QEMU to crash. To mitigate you can use the environment variable to disable that option globally: add `SERENITY_DISABLE_GDB_SOCKET=1` in `/etc/environment`
```

Comment:
I'm not sure what you mean by "does not work". I've used `./Meta/serenity.sh gdb` fine before I'm WSL with Windows qemu. Was there a specific version of qemu you were seeing crash with this option?

Reply:
@ADKaster  I mean it fails to start qemu with the error described. I'm using latest from here - https://qemu.weilnetz.de/w64/ , specifically this one "qemu-w64-setup-20210825.exe ".  It launched fine for first time after toolchain rebuild and "meta/serenity.sh run" commands. After that I've closed QEMU and all subsequent starts were failing with the error  `qemu-system-i386.exe: -s: Failed to bind socket: Input/output error`.   Winning win10
This was the fix for me. 

For more info: 
Host OS Name:                   Microsoft Windows 10 Pro N
Host OS Version:                10.0.19042 N/A Build 19042
Ubuntu WSL version: 5.10.16.3-microsoft-standard-WSL2